egs/aishell/ASR/transducer_stateless/joiner.py

```diff
@@ -19,10 +19,12 @@
 
 
 class Joiner(nn.Module):
-    def __init__(self, input_dim: int, output_dim: int):
+    def __init__(self, input_dim: int, inner_dim: int, output_dim: int):
         super().__init__()
 
-        self.output_linear = nn.Linear(input_dim, output_dim)
+        self.output_linear = nn.Sequential(
+            nn.Linear(input_dim, inner_dim), nn.Linear(inner_dim, output_dim)
```

Comment:
There are no non-linear layers between the two `nn.Linear` layers. Is there a reason to introduce an extra `nn.Linear` layer instead of using a single `nn.Linear` layer here?

Reply:
Using two `nn.Linear` here to reduce the num of parameters.